diff --git a/node_modules/react-native-ui-lib/src/commons/Constants.js b/node_modules/react-native-ui-lib/src/commons/Constants.js
index 795f5d3..b7e7d3a 100644
--- a/node_modules/react-native-ui-lib/src/commons/Constants.js
+++ b/node_modules/react-native-ui-lib/src/commons/Constants.js
@@ -1,4 +1,4 @@
-import { Platform, Dimensions, NativeModules, I18nManager, AccessibilityInfo, StatusBar } from 'react-native';
+import { Platform, Dimensions, I18nManager, AccessibilityInfo } from 'react-native';
 export let orientations = /*#__PURE__*/function (orientations) {
   orientations["PORTRAIT"] = "portrait";
   orientations["LANDSCAPE"] = "landscape";
@@ -21,14 +21,7 @@ let defaultMargin = 0;
 const isSubWindow = windowWidth < screenWidth;
 isTablet = Platform.OS === 'ios' && Platform.isPad || getAspectRatio() < 1.6 && Math.max(screenWidth, screenHeight) >= 900;
 function setStatusBarHeight() {
-  const {
-    StatusBarManager
-  } = NativeModules;
-  statusBarHeight = (StatusBar.currentHeight ?? StatusBarManager?.HEIGHT) || 0;
-  if (isIOS && StatusBarManager) {
-    // override guesstimate height with the actual height from StatusBarManager
-    StatusBarManager.getHeight(data => statusBarHeight = data.height);
-  }
+  statusBarHeight = 0;
 }
 function getAspectRatio() {
   return screenWidth < screenHeight ? screenHeight / screenWidth : screenWidth / screenHeight;
